baselines/dom.generated.d.ts

```diff
@@ -29370,6 +29370,12 @@ interface SVGElement extends Element, ElementCSSInlineStyle, GlobalEventHandlers
      * [MDN Reference](https://developer.mozilla.org/docs/Web/API/SVGElement/viewportElement)
      */
     readonly viewportElement: SVGElement | null;
+    /**
+     * The addEventListener() method of the EventTarget interface sets up a function that will be called whenever the specified event is delivered to the target.
+     *
+     * [MDN Reference](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)
+     */
+    addEventListener(type: string, listener: ((event: Event) => void) | ((event: UIEvent) => void)): void;
```

Comment:
This UIEvent thing is not directly related to onerror, this should be in a separate patch with separate discussion. I don't think we should do this at all, because why UIEvent specifically and not others? Listing all other events is not viable, but picking UIEvent is rather arbitrary.